lib/fluent/plugin/s3_compressor_gzip_command.rb

```diff
@@ -4,6 +4,7 @@ class GzipCommandCompressor < Compressor
       S3Output.register_compressor('gzip_command', self)
 
       config_param :command_parameter, :string, :default => ''
+      config_param :customer_tmp_dir, :string, :default => Dir.tmpdir
```

Comment:
Good idea, I modified it, could you please check it again? Thank you very much. @repeatedly